src/SFA.DAS.EmployerIncentives/Models/EmployerIncentives/CreateIncentiveApplicationRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Models.EmployerIncentives
+{
+    public class PostCreateIncentiveApplicationRequest : IPostApiRequest
```

Comment:
Should this not be in the InnerApi Request namespace? Also missing unit test coverage